src/Features/Core/Portable/Completion/CompletionRules.cs

```diff
@@ -41,6 +44,21 @@ protected PatternMatcher GetPatternMatcher(string value)
             }
         }
 
+        protected PatternMatcher GetEnUSPatternMatcher(string value)
```

Comment:
I would call this this FallbackPatternMatcher, if we change our lookup rules here, this API name would need to change to match the new behavior.